.github/workflows/dapr-standalone-validation.yml

```diff
@@ -59,7 +59,7 @@ jobs:
       - name: Validate sidecar's basic resource utilization
         env:
           SECONDS_FOR_PROCESS_TO_RUN: 30
-          LIMIT_DELTA_BINARY_SIZE: 1024  # KB
+          LIMIT_DELTA_BINARY_SIZE: 5120  # KB (5 MB)
```

Comment:
I thought we discussed doing 7? Esp since we're less concerned with the size of the binary itself